staging/src/k8s.io/kubectl/docs/book/pages/extending_kubectl/discovering_plugins.md

```diff
@@ -3,13 +3,13 @@
 {% endpanel %}
 
 {% panel style="info", title="TL;DR" %}
-- [krew.dev](https://github.com/kubernetes-sigs/krew/#installation) is a kubernetes sub-project to discover and manage plugins
+- [sigs.k8s.io/krew](https://sigs.k8s.io/krew/#installation) is a kubernetes sub-project to discover and manage plugins
```

Comment:
I have an outstanding PR that'll render this broken.

https://krew.sigs.k8s.io/docs/user-guide/setup/install/ might be better.

Reply:
I've adapted the link so that it points to our new project site.